src/components/app/MenuButtons/Publish.js

```diff
@@ -99,7 +100,7 @@ class MenuButtonsPublishImpl extends React.PureComponent<PublishProps> {
           onChange={toggle}
           checked={checkedSharingOptions[slug]}
         />
-        {label}
+        <Localized id={label} />
```

Comment:
nit: can you please rename `label` to `labelL10nId` so that it's clear this is a localization ID? Thanks